### ISSUE TITLE
map cmd + ; to ctrl + b

### ISSUE THREAD

**Issue by jasonoverby** [zeit/hyper#3480]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  macOS Sierra 10.12.6
- **Hyper.app version**: Hyper 3.0.0-canary.8 (canary)
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/jasonoverby/45b8679850ad149d36e5a5a1a90aa5e2
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A>
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue
I love Hyper, and I am hoping to replace iterm with it.  I am having difficulty, however, with tmux bindings.  Specifically, I am hoping to map my ideal tmux prefix `cmd-;` to the standard tmux prefix (`ctrl-b`).  I have adjusted the keymaps in my preferences, but this doesn't seem to work.  What am I missing?  Thank you!!

**Comment by ppot** [comment]
Yeah. The Keymap I introduced is to bind keymap to hyper commands. Not underlaying tmux binding. 
On your config : 
```
  keymaps: {
// Example
// 'window:devtools': 'cmd+alt+o',
// '0x02':'command+;',
'ctrl+b':'command+;',
  },
```

You can clearly see that: `ctrl+b` isn't a https://github.com/zeit/hyper/blob/canary/app/keymaps/darwin.json command. I can see it's the default prefix of tmux. However tmux dosen't support the `command` meta key. Maybe @chabou can point you on how to send a command to hyper that send `C-b` to hyper.

**Comment by jasonoverby** [comment]
Thanks, @ppot & that makes sense.
